src/assets/lang/en.json

```diff
@@ -33,6 +33,7 @@
 	"rows": "Rows",
 	"success": "Success",
 	"table": "Insert Table",
+	"video": "Insert Video",
```

Comment:
I think we're supposed to add the keys to `src/assets/lang/language.json`, not this file.

Reply:
Yes, you're right.